packages/firestore/test/unit/local/mutation_queue.test.ts

```diff
@@ -433,7 +434,30 @@ function genericMutationQueueTests(): void {
     const matches = await mutationQueue.getAllMutationBatchesAffectingDocumentKey(
       key('foo/bar')
     );
-    expect(matches.length).to.deep.equal(expected.length);
```

Comment:
Forgot to mention: I noticed that `expectEqualArrays` checks for equal length, so this line seems redundant.